astroquery/__init__.py

```diff
@@ -27,3 +27,26 @@ def _get_bibtex():
 
 
 __citation__ = __bibtex__ = _get_bibtex()
+
+# backward compatibility: retain the flat namespace as of May 2019 at least
+# until we deprecate it
+
+# categories of data type
+from atomic_molecular_databases import atomic, hitran, jplspec, nist, splatalogue, vamdc
```

Comment:
I don't think these changes are right, currently the modules  are not available by importing astroquery.

Reply:
also, these imports should be relative imports otherwise `ModuleNotFoundError` is raised